docs/serviceregistry/manuals/temperature-http-demo.md

```diff
@@ -0,0 +1,846 @@
+
+Since we know that the provider name is _temperature-provider2_, the service definition name is alert-service, and the version was 1.0.0, we will send a lookup request with these filters. We will set the _verbose_ flag to false, because we only need the service instance ID here, all the other additional data is unnecessary.
+
+Note that our case, the service discovery policy is set to _OPEN_. If the discovery policy is _RESTRICTED_, we will only retrieve the services that have the  metadata key _unrestricted-discovery_, and this is set to _true_.
+
+The lookup request looks like this:
+
+~~~
+curl -X 'POST' \
+  'http://localhost:8443/serviceregistry/service-discovery/lookup?verbose=false' \
+  -H 'accept: application/json' \
+  -H 'Authorization: Bearer SYSTEM//temperature-provider2' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "instanceIds": [
+  ],
+  "providerNames": [
+    "temperature-provider2"
+  ],
+  "serviceDefinitionNames": [
+    "alert-service"
+  ],
+  "versions": [
+    "1.0.0"
+  ],
+  "alivesAt": "",
+  "metadataRequirementsList": [
+  ],
+  "interfaceTemplateNames": [
+  ],
+  "interfacePropertyRequirementsList": [
+  ],
+  "policies": [
+  ]
+}'
+~~~
+
+The lookup response:
+
+~~~
+{
+  "entries": [
+    {
+      "instanceId": "temperature-provider2::alert-service::1.0.0",
+      "provider": {
+        "name": "temperature-provider2",
+        "metadata": {
+          "scales": [
+            "Kelvin",
+            "Celsius"
+          ],
+          "location": {
+            "side": "North",
+            "block": 2
+          },
+          "indoor": true
+        },
+        "version": "1.0.0",
+        "createdAt": "2024-11-08T10:21:11Z",
+        "updatedAt": "2024-11-08T10:21:11Z"
+      },
+      "serviceDefinition": {
+        "name": "alert-service",
+        "createdAt": "2024-11-08T15:23:10Z",
+        "updatedAt": "2024-11-08T15:23:10Z"
+      },
+      "version": "1.0.0",
+      "expiresAt": "2025-01-01T00:00:00Z",
+      "metadata": {
+        "max-delay": {
+          "value": 15,
+          "unit": "sec"
+        }
+      },
+      "interfaces": [
+        {
+          "templateName": "generic-http",
+          "protocol": "http",
+          "policy": "NONE",
+          "properties": {
+            "accessAddresses": [
+              "192.168.56.116",
+              "tp2.greenhouse.com"
+            ],
+            "accessPort": 8080,
+            "operations": {
+              "subscribe": {
+                "path": "/subscribe",
+                "method": "GET"
+              },
+              "unsubscribe": {
+                "path": "/unsubscribe",
+                "method": "GET"
+              },
+              "set-threshold": {
+                "path": "/threshold",
+                "method": "POST"
```

Comment:
Don't forget to change the methods (after you changed them in the registration part)